src/Compilers/CSharp/Portable/Symbols/Source/SourcePropertySymbolBase.cs

```diff
@@ -1321,7 +1321,7 @@ protected override void DecodeWellKnownAttributeImpl(ref DecodeWellKnownAttribut
                 }
                 else
                 {
-                    diagnostics.Add(ErrorCode.ERR_UnscopedRefAttributeUnsupportedTarget, arguments.AttributeSyntaxOpt.Location);
+                    diagnostics.Add(ErrorCode.ERR_UnscopedRefAttributeUnsupportedMethodTarget, arguments.AttributeSyntaxOpt.Location);
```

Comment:
>ERR_UnscopedRefAttributeUnsupportedMethodTarget [](http://example.com/codeflow?start=46&length=47)

The wording is probably going to sound strange for a property. It excludes properties.